build(client): Changelog generation for minor release 2.12.0 (#23342)
microsoft/FluidFramework: packages/loader/container-loader/CHANGELOG.md

# @fluidframework/container-loader

## 2.12.0

### Minor Changes

-   New APIs to create and load containers without using the Loader object ([#22902](https://github.com/microsoft/FluidFramework/pull/22902)) [51a17289c6](https://github.com/microsoft/FluidFramework/commit/51a17289c683ff6666e496878cb6660d21759b16)

    #### Overview

    Provide standalone APIs to create and load containers instead of using the Loader object to do so. Earlier hosts were
    supposed to create the Loader object first and then call methods on it to create and load containers. Now they can just
    utilize these APIs directly and get rid of the Loader object.

    ##### Use `createDetachedContainer` to create a detached container

    ```typescript
    export async function createDetachedContainer(
    	createDetachedContainerProps: ICreateDetachedContainerProps,
    ): Promise<IContainer> {}
    ```

>     `ICreateDetachedContainerProps` are the properties that needs to be supplied to the above API which contains props like

Reviewer: ```suggestion
    `ICreateDetachedContainerProps` are the properties that need to be supplied to the above API and include props like
```

    URL Resolver, IDocumentServiceFactory, etc., which were previously used to create the `Loader` object.

    ##### Use `loadExistingContainer` to load an existing container

    ```typescript
    export async function loadExistingContainer(
    	loadExistingContainerProps: ILoadExistingContainerProps,
    ): Promise<IContainer> {}
    ```

>     `ILoadExistingContainerProps` are the properties that needs to be supplied to the above API which contains props like

Reviewer: ```suggestion
    `ILoadExistingContainerProps` are the properties that need to be supplied to the above API and include props like
```

    URL Resolver, IDocumentServiceFactory, etc., which were earlier used to create the `Loader` object.

    ##### Use `rehydrateDetachedContainer` to create a detached container from a serializedState of another container

    ```typescript
    export async function rehydrateDetachedContainer(
    	rehydrateDetachedContainerProps: IRehydrateDetachedContainerProps,
    ): Promise<IContainer> {}
    ```

>     `IRehydrateDetachedContainerProps` are the properties that needs to be supplied to the above API which contains props like

Reviewer: ```suggestion
    `IRehydrateDetachedContainerProps` are the properties that need to be supplied to the above API and include props like
```
